.travis.yml

```diff
@@ -1,10 +1,21 @@
-sudo: required
+sudo: false
 language: c
 os:
   - linux
 branches:
   except:
     - nightly
+addons:
+    apt_packages:
+        - xclip
+        - gdb
+        - valgrind
+        - libtool
+        - gcc-multilib
```

Comment:
In Arch Linux, at least, it is not installed by default and requires sourcing a repo. Although multilib does work for x86_64